Autopilot/Inc/altimeter.hpp

```diff
@@ -35,4 +38,24 @@ class Altimeter{
         virtual void GetResult(AltimeterData_t *Data) = 0; //
 };
 
+class MS5637 : public Altimeter {
+    public:
+        MS5637(const MS5637*) = delete; //Apparently if you try to copy a singleton this will give you errors?
+        static MS5637* GetInstance();
+        void Init();
+        void Begin_Measuring();
+        void GetResult(AltimeterData_t *Data);
+    private:
+        MS5637() {} //Constructor can never be called muwhahaha
```

Comment:
hmmm ya probs would be good

Reply:
Actually I think a reference to the instance should be okay, as long as you don't do something like
`MS5637 reference = MS5637::GetInstance->function()`
instead of
`MS5637* reference`

should be okay since line 43 takes care of that edge case and will not compile lol